packages/cli/src/databases/repositories/sharedCredentials.repository.ts

```diff
@@ -36,27 +36,23 @@ export class SharedCredentialsRepository extends Repository<SharedCredentials> {
 		return await this.update({ userId: Not(user.id), role: 'credential:owner' }, { user });
 	}
 
-	/**
-	 * Get the IDs of all credentials owned by or shared with a user.
-	 */
-	async getAccessibleCredentials(userId: string) {
-		const sharings = await this.find({
-			where: {
-				userId,
-				role: In(['credential:owner', 'credential:user']),
-			},
-		});
-
-		return sharings.map((s) => s.credentialsId);
+	/** Get the IDs of all credentials owned by or shared with a user */
+	async getAccessibleCredentialIds(userIds: string | string[]) {
+		return await this.getCredentialIdsForUserAndRole(userIds, [
+			'credential:owner',
+			'credential:user',
+		]);
 	}
 
-	async findOwnedSharings(userIds: string[]) {
-		return await this.find({
+	async getCredentialIdsForUserAndRole(userIds: string | string[], roles: CredentialSharingRole[]) {
+		if (!Array.isArray(userIds)) userIds = [userIds];
```

Comment:
Should we add more code for a minor convenience? I'd favor consistency in input types.